example/basic_widget.html

```diff
@@ -122,7 +122,7 @@
         </form>
       </div>
 
-      <div class="hidden" data-screen data-login-confirm="otp">
+      <div class="hidden" data-screen data-login-confirm="totp">
```

Comment:
Maybe missing something, but I think we are mapping it to `otp` here:
https://github.com/auth0/auth0-mfa-api/blob/master/lib/models/device_account/device_account.model.js#L347

Reply:
great catch! you are right, the original version of the method used to return totp but since we use otp everywhere else I changed it server side.